packages/api/tests/models/rules/test-rules-model.js

```diff
@@ -196,6 +196,20 @@ test.serial('Creating an invalid rule does not create workflow triggers', async
   }
 });
 
+test('Creating a rule without a createdAt or updatedAt date assigns a value to createdAt and updatedAt', async (t) =>{
```

Comment:
I think the new functionality that we want to test is that if we **do provide a createdAt/updatedAt** to `rule.create`, then it is used. This test seems confirming the existing behavior before your changes - that createdAt/updatedAt always get populated, but it doesn't test what values are used

Reply:
[`15f822f` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/15f822f315e854df2abcce13ecf29bb81f5cfd43)

I am wondering if my checks are overkill for `updatedAt`. I want to ensure that the value passed in for `updatedAt` in fact gets updated to another, more current date

Reply:
[`77f5f85` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/77f5f859682d26face2bd271929a656456819d8f)

Actually check out that diff instead